docs/apache-airflow/concepts/timetable.rst

```diff
@@ -67,6 +67,28 @@ cron expression to the ``schedule_interval`` parameter of a DAG as described in
     def example_dag():
         pass
 
+CronTriggerTimetable
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Another timetable which accepts a cron expression. It behaves more compliant with standard cron than ``CronDataIntervalTimetable``.
```

Comment:
Let’s re-order these to

1. CronTriggerTimetable
2. DeltaDataIntervalTimetable
3. CronDataIntervalTimetable

since it is probably the order we want to recommend these to newcomers.

Reply:
Sounds good. Re-ordered by Thanks for your prompt review! Made it shorter by https://github.com/apache/airflow/pull/23662/commits/74723e9ef2faa92d5cb389238ef5f5850adbe179 (same commit)